### ISSUE TITLE
Query for nested values, not documented.

### ISSUE THREAD

**Issue by daveybrown** [graphql-compose/graphql-compose-elasticsearch#68]
Hi,

I'm hitting a brick wall here, so any help would be appreciated. I have data like this:

    "_source": {
      "ItemName": "test title 4",
      "ItemBody": "The description",
      "ItemAttributes": {
        "colour": [
          "blue",
          "red"
        ],
        "size": [
          "s"
          "m"
          "l"
        ]
      }
    },

Is there a way to query this data, without using the dot notation, like I've tried (below). The dot is not a vilid field name, so the query doesn't work. 

    search(
      body: {
        query: {
            match : { ItemAttributes.colour : "blue"  } 
        }
      }
    )

Thanks in advance. Davey

**Comment by vojtatranta** [comment]
Yeah, this does not work it would be  super cool to make ti work!
